tuneme/static/css/styles.css

```diff
@@ -108,7 +108,7 @@ label {font:normal 15px 'Quicksand', Arial, Helvetica, sans-serif;}
 
     /* Radios */
     input[type='radio'] {width:auto; padding:0px; margin:0 5px 0 0; font-size:15px;}
-    .radios label { font-size:15px; min-width:75px; font-weight:normal;}
+    .radios label { font-size:15px; min-width:75px; font-weight:normal; display: inline-flex; width: 90%;}
```

Comment:
I doubt the support for `flex` on mobile devices. Can you use `inline-block` instead?